test/e2e/density.go

```diff
@@ -40,8 +40,17 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-// NodeStartupThreshold is a rough estimate of the time allocated for a pod to start on a node.
-const NodeStartupThreshold = 4 * time.Second
+const (
+	// NodeStartupThreshold is a rough estimate of the time allocated for a pod to start on a node.
+	NodeStartupThreshold = 4 * time.Second
+
+	// minPodStartupThroughput is the allowed minimum pod startup throughput
+	// per node (pods per minute). Note that latency it takes for the nodes to
+	// report the pods are running is usually the bottleneck in pod startup
+	// because when the density (pods per node) is high. We use a rather
+	// conservative number to avoid premature timeout while waiting.
+	minPodStartupThroughput = 15.0
```

Comment:
Since is not really pod startup throughput - this is per-node pod startup throughput. Can you rename it to reflect it, e.g.:
minPerNodePodStartupThroughput
or sth like that?